### PR TITLE
add partner_area and web_google_maps_drawing for Odoo13

### PR DESCRIPTION
**Pull request by desdelinux**

### CODE REVIEW

File: web_google_maps_drawing/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Web Google Maps Drawing',
+    'version': '12.0.1.0.0',
```
Comment (gityopie) [inline web_google_maps_drawing/__manifest__.py:4]: Could you please update the version number?

File: web_google_maps_drawing/data/google_maps_library.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <function model="ir.config_parameter" name="set_param" eval="('google.maps_libraries', 'geometry,places,drawing')"/>
```
Comment (gityopie) [inline web_google_maps_drawing/data/google_maps_library.xml:4]: The config param `key` should be `web_google_maps.maps_libraries`

File: web_google_maps_drawing/example/partner_area/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Partner Area (Google Maps Drawing demo implementation)',
+    'version': '12.0.1.0.1',
```
Comment (gityopie) [inline web_google_maps_drawing/example/partner_area/__manifest__.py:4]: same for the version number :)

File: partner_area/views/res_partner.xml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[1]" position="attributes">
+                <attribute name="autofocus"/>
+            </xpath>
+            <xpath expr="//notebook/page[1]" position="before">
+                <page string="Area" autofocus="autofocus">
+                    <field name="shape_line_ids" mode="map">
+                        <map library="drawing" string="Shape">
```
Comment (gityopie) [inline partner_area/views/res_partner.xml:14]: The view `map` has been changed to `google_map`. On the enterprise, Odoo has this view type

File: partner_area/views/res_partner_area.xml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                <field name="partner_id"/>
+                <field name="shape_name"/>
+                <field name="shape_type"/>
+                <field name="shape_area"/>
+                <field name="shape_radius"/>
+                <field name="shape_description"/>
+            </tree>
+        </field>
+    </record>
+    <!--record model="ir.ui.view" id="view_res_partner_area_kanban">
```
Comment (gityopie) [inline partner_area/views/res_partner_area.xml:49]: If this view is no longer necessary, please remove it :)

File: web_google_maps_drawing/example/partner_area/views/res_partner.xml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[1]" position="attributes">
+                <attribute name="autofocus"/>
+            </xpath>
+            <xpath expr="//notebook/page[1]" position="before">
+                <page string="Area" autofocus="autofocus">
+                    <field name="shape_line_ids" mode="map">
+                        <map library="drawing" string="Shape">
```
Comment (gityopie) [inline web_google_maps_drawing/example/partner_area/views/res_partner.xml:14]: should be `google_map` not `map` 
and the other views that still using `map`

**Review by gityopie — CHANGES_REQUESTED** [review]
Hi @crece365 
Several points to check on your side :)

Thanks for the PR!

**Comment by macagua** [comment]
@crece365 please, check out the @gityopie review and make these changes!!!

**Pull request closed without merging**